### PR TITLE
Touch up

### PR DESCRIPTION
**Pull request by quanvo87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
 ![Apache 2](https://img.shields.io/badge/license-Apache2-blue.svg?style=flat)
 &nbsp;[![Slack Status](http://swift-at-ibm-slack.mybluemix.net/badge.svg)](http://swift-at-ibm-slack.mybluemix.net/)
+[![codecov](https://codecov.io/gh/IBM-Swift/Kitura-WebSocket/branch/master/graph/badge.svg)](https://codecov.io/gh/IBM-Swift/Kitura-WebSocket)
 
 ## Summary
 
 Kitura-WebSocket provides Kitura based servers the ability to receive and send messages to clients using the WebSocket
 protocol (RFC 6455). It is compatible with a variety of WebSocket clients, including:
-- The builtin WebSocket support in the Chrome, FireFox, and Safari browsers
+- The built in WebSocket support in the Chrome, FireFox, and Safari browsers
```
Comment (youming-lin) [inline README.md]: Technically one word, `built-in`, instead of two words, `built in`.

File: Sources/KituraWebSocket/WSServerRequest.swift [hunk 2]
```diff
@@ -41,9 +41,8 @@ class WSServerRequest: ServerRequest {
     private var urlc: URLComponents?
     
     /// The URL from the request as URLComponents
-    /// URLComponents has a memory leak on linux as of swift 3.0.1. Use 'urlURL' instead
-    @available(*, deprecated, message:
-    "URLComponents has a memory leak on linux as of swift 3.0.1. use 'urlURL' instead")
+    /// URLComponents has a memory leak on Linux as of Swift 3.0.1. Use 'urlURL' instead.
+    /// This is fixed in Swift 3.1.
     public var urlComponents: URLComponents {
```
Comment (youming-lin) [inline Sources/KituraWebSocket/WSServerRequest.swift]: What happens to `urlURL` if `urlComponents` is no longer deprecated?

**Comment by quanvo87** [comment]
@youming-lin Latest commit addresses review comments

File: Sources/KituraWebSocket/WSServerRequest.swift [hunk 3]
```diff
@@ -38,23 +38,12 @@ class WSServerRequest: ServerRequest {
...
-        self.urlc = urlc
-        return urlc
-    }
-    
     /// The URL from the request
     let urlURL: URL
-    
+
+    /// The URL from the request as URLComponents
+    public var urlComponents: URLComponents
```
Comment (youming-lin) [inline Sources/KituraWebSocket/WSServerRequest.swift]: Should be `public let`. It wasn't a settable computed property before.

**Pull request merged**